app/src/pages/inside/testItemPage/notFound.jsx

```diff
@@ -5,19 +5,25 @@ import styles from './notFound.scss';
 
 const cx = classNames.bind(styles);
 const messages = defineMessages({
-  notFound: {
-    id: 'LaunchesPage.notFound',
+  launchNotFound: {
+    id: 'LaunchesPage.launchNotFound',
     defaultMessage: 'Launch is not found',
   },
   notFoundDescription: {
```

Comment:
It looks like you forgot translate this message and customise it for both of use case.

Reply:
It's the same behavior as in v4